cairo/ethereum/crypto/alt_bn128.cairo

```diff
@@ -45,235 +45,340 @@ func FROBENIUS_COEFFICIENTS() -> TupleBNF12 {
 
     tempvar frob_coeff_0 = BNF12(
         new BNF12Struct(
-            UInt384(1, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
-            UInt384(0, 0, 0, 0),
+            U384(new UInt384(1, 0, 0, 0)),
+            U384(new UInt384(0, 0, 0, 0)),
```

Comment:
for all U384_ZERO values we should have a label, like we do with `U256_ZERO`, and use `let (u384_zero) = get_label_location(U256_ZERO);` then i think we can do

`U384(cast(u384_zero, UInt384*))`